### DISCUSSION TITLE
Account restricted for violation of terms of service?

### DISCUSSION THREAD

**Discussion by jenvigo** [codewars/codewars.com#2860] [Q&A] [answered]
Hi, yeah. I am guilty. I had curiosity of what happens if you just paste a random answer from GitHub for a 1kyu test. 
Now I know  :) and I apologize for "cheating" but I guess the previous 130 katas that I did before shouldn't pay for this mistake. 

Can I have my account fully back, please? Maybe helping with anything in the platform. I would love to collaborate.

**Comment by Blind4Basics** [comment]
Hi,

130 kata!? Where so? Would you have a _third_ account we don't know about yet? That would already be locked as well?? Which one are we supposed to unlock, exactly?

**Reply by jenvigo** [reply]
Hi @Blind4Basics Thanks for the quick response. My regular account is jenThepilot. I have two accounts as far as I know. The reason is that I like to create a new gmail account everytime that I start a new job so the things of those experiences don't get mixed with each other. And when I log in codewars sometimes I forgot passwords and log in with the current session. If that is really against the platform policy then I apologize for it. 

I am not a hardcode user of it and I don't care too much about the rank. I accept a penalty of points or whatever you consider fair.

**Reply by jenvigo** [reply]
and yes...in that account I finished 132 katas according to the stats. ONE of them copy/pasting.

**Reply by jenvigo** [reply]
Currently, the only feature that I am missing is being able to compare my solutions against the rest of the community, which is the feature that I like the most.

**Reply by Blind4Basics** [reply]
I don't understand:
- jenThepilot has 36 completed kata, not 132
- jenvigo has 3, not 132
So, what account are we supposed to unlock, here?

**Comment by jenvigo** [comment, marked as answer]
My bad, I was reading the stats wrongly. 
![image](https://user-images.githubusercontent.com/8104456/201313689-106fd0dc-fd22-4eb4-8ed8-e7beba1fc4c8.png)
I thought that the amount of katas was that bar over there. I see that it's indeed 36 and not 132.

The account to be unlocked is jenThepilot, if possible. Thanks in advance.

**Reply by Blind4Basics** [reply]
done

**Reply by jenvigo** [reply]
Great, I appreciate it.
